packages/taquito-local-forging/src/codec.ts

```diff
@@ -57,6 +57,18 @@ export const pkhDecoder = (val: Uint8ArrayConsumer) => {
   }
 };
 
+export const pkhsDecoder = (val: Uint8ArrayConsumer) => {
```

Comment:
what would the `s` be in this case in `pkhs`

Reply:
same as the other comment inherit the naming consistency from `proposalsDecoder`, I think `s` means plural

Reply:
I think for proposals it's because the operation itself is called `Proposals` instead of `proposal`.

I think we can remove the `S` in this case, but that's just personal preference.

Reply:
there are `proposalDecoder` and `proposalsDecoder`. There is already `pkhDecoder` will we consider `pkhArrayDecoder`?

Reply:
yeah let's do pkhArrayDecoder then, seems more informative than just using `s`.